opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1016,12 +1014,15 @@ namespace Opm {
                 const unsigned cellIdx = elemCtx.globalSpaceIndex(/*spaceIdx=*/0, /*timeIdx=*/0);
                 const auto& intQuants = elemCtx.intensiveQuantities(/*spaceIdx=*/0, /*timeIdx=*/0);
                 const auto& fs = intQuants.fluidState();
+                const double pv =
+                    ebosSimulator_.model().dofTotalVolume(cellIdx)
+                    * ebosSimulator_.problem().porosity(cellIdx);
```

Comment:
> Why not keep using geo_.poreVolume()?

because its quite easy to switch away from it, with `geo_` it is much more difficult to also include rock compressibility if this turns out to be the Right Thing (TM) to do, it makes the parallel code less explicit (IMO) because it needs use non-standard mechanisms, and it forces the legacy object to be kept around even if it is not used by anything else...